pkgdown/_pkgdown.yml

```diff
@@ -43,26 +56,35 @@ reference:
   - dm_get_colors
   - dm_get_available_colors
 
+- title: Zooming to a table of a `dm`
+  contents:
+  - dm_zoom_to_tbl
```

Comment:
Maybe:

```suggestion
  - dm_zoom_to
```

?

Reply:
ok, changed the name from `dm_zoom_to_tbl` to `dm_zoom_to`